engines/wintermute/ad/ad_game.cpp

```diff
@@ -1697,6 +1697,19 @@ Wintermute::TShadowType Wintermute::AdGame::getMaxShadowType(Wintermute::BaseObj
 
 	return MIN(ret, _scene->_maxShadowType);
 }
+
+bool Wintermute::AdGame::getFogParams(bool *fogEnabled, uint32 *fogColor, float *fogStart, float *fogEnd) {
```

Comment:
Maybe declare a struct for this purpose, and return that instead of all these pointers?

Reply:
This struct would probably be best declared together with the 3d renderer, which does not exist in this branch. So I would prefer to add this later.

Reply:
Agreed